src/czml3/properties.py

```diff
@@ -291,17 +344,23 @@ def __init__(
         style=LabelStyles.FILL,
         scale=None,
         showBackground=None,
+        backgroundColor=None,
         horizontalOrigin=None,
         verticalOrigin=None,
+        fillColor=None,
+        outlineColor=None,
         outlineWidth=1.0,
     ):
         self._show = show
         self._text = text
         self._style = style
         self._scale = scale
         self._show_background = showBackground
+        self._backgroundColor = backgroundColor
```

Comment:
For consistency, let's keep using `._background_color` for private properties (keeping `backgroundColor` for the public API)